infra/scripts/install_requirements_2_5.txt

```diff
@@ -4,3 +4,6 @@ wheel==0.43.0
 circle_schema
 
 torch==2.5.0
+cffi==1.17.1
+packaging==25.0
+pyyaml==6.0.2
```

Comment:
Using latest release versions.

Reply:
@dayo09 Not related this PR but on second thought, `pyyaml` doesn't need to be inside setup.py's `install_requires` and install requirement. Isn't it more like a test requirement?

Reply:
@mhs4670go I agree 👍